common/crypto/attestation-api/ocalls/attestation-ocalls.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <sgx_uae_epid.h>
+#include "error.h"
+#include "fpc-types.h"
+#include "logging.h"
+#include "sgx_quote.h"
+
+void ocall_init_quote(uint8_t* target, uint32_t target_len, uint8_t* egid, uint32_t egid_len)
```

Comment:
It's a minor thing. The cost is amortized over the lifetime of the enclave.

Reply:
it's ultimately more a code complexity issue (i.e., if enclave-internal sdk function for this would have existed, we could have saved ourselves the edl and untrusted functions) . But as mentioend, there is nothing we really can do as the SDK doesn't off the enclave-internal variant of the functions ...